internal/output/outputters.go

```diff
@@ -0,0 +1,42 @@
+package output
+
+import "strings"
+
+type MultipleOutputter struct {
```

Comment:
I could move the interface definition in here as well if that would make it easier to reason about. I left it in `output.go` to keep that as the one place where we define types that are implemented in other places, but it's not a big deal either way.